histomicstk/segmentation/label/trace_boundaries.pyx

```diff
@@ -0,0 +1,21 @@
+"""
+Cython source code: trace_boundaries
+"""
+import cython
+
+import numpy as np
+cimport numpy as np
+
+from libcpp.vector cimport vector
+
+cdef extern from "trace_boundaries_opt.h":
+  vector[vector[vector[int]]] trace_boundary(vector[vector[int]] label, int)
+
+
+def trace_boundaries(np.ndarray[int, ndim=2, mode="c"] imLabel not None, connectivity=4):
+
+    cdef vector[vector[vector[int]]] output
+
+    output = trace_boundary(imLabel, connectivity)
```

Comment:
Added a padding for each object using min-max of coordinates, and then finally added window offset [here](https://github.com/DigitalSlideArchive/HistomicsTK/pull/212/files#diff-22e72ae0b92c8b76a2c9bd57f2ff509bR85) Would it be better using it out of the function? If it is, I will add them before the function and then removed the previous work.